indico/modules/events/contributions/controllers/management.py

```diff
@@ -679,3 +680,10 @@ def _process_POST(self):
         self.subcontrib.references.append(reference)
         db.session.flush()
         return self.jsonify_reference(reference)
+
+
+class RHCloneContribution(RHManageContributionBase):
+    def _process(self):
+        clone_contribution(self.event, self.contrib, False)
```

Comment:
I would use a kwarg for the last one, or just omit it and set a default value of False.